src/Silk.Core/Commands/Tests/TagCommand.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using DSharpPlus.Interactivity.Enums;
+using DSharpPlus.Interactivity.Extensions;
+using MediatR;
+using Silk.Core.Services;
+using Silk.Core.Utilities.HelpFormatter;
+using Silk.Data.MediatR;
+using Silk.Data.Models;
+using Silk.Extensions;
+
+namespace Silk.Core.Commands.Tests
+{
+    [RequireGuild]
+    [Group("tag")]
+    [Category(Categories.Server)]
+    public class TagCommand : BaseCommandModule
+    {
+        private readonly IMediator _mediator;
+        private readonly TagService _tagService;
+        private readonly string[] _reservedWords = new[]
+        {
+            "create", "update", "delete",
+            "alias", "info", "claim"
+        };
```

Comment:
Instead of reserving words, couldn't you just make the command priority of the get command lower than subcommands?

Reply:
Theoretically, though that could still error out commands on both ends; people that want to actually do something with a tag, and someone who wants to use a specific tag.

Reply:
Even if possible, this would result in a tag with the name of a reserved word to never work.

i.e. tag create with no parameters -> error, tag create with parameters -> tag created

A tag called `create` would never be displayed this way.

Reply:
I do agree with Natsumi.